core/consensus/grandpa/impl/voting_round_impl.cpp

```diff
@@ -791,27 +796,28 @@ namespace kagome::consensus::grandpa {
                    signed_precommit.id.toHex());
           continue;
         }
-        if (env_->hasAncestry(vote.hash, signed_precommit.getBlockHash())) {
+        if (has_ancestry(justification.block_info,
+                         signed_precommit.getBlockInfo())) {
           total_weight += weight_opt.value();
         } else {
           SL_DEBUG(logger_,
                    "Vote does not have ancestry with target block: "
                    "vote={} target={}",
-                   vote.hash,
+                   justification.block_info.hash,
                    signed_precommit.getBlockHash());
         }
 
       } else if (equivocators.emplace(signed_precommit.id).second) {
         // Detected equivocation
-        if (env_->hasAncestry(vote.hash, it->second)) {
+        if (has_ancestry(justification.block_info, it->second)) {
```

Comment:
May be rename `has_ancestry` to `hasAncestry` to follow name convantion?

Reply:
It's lambda variable, not function